components/badge/index.tsx

```diff
@@ -133,7 +133,7 @@ const Badge: CompoundedComponent = ({
   const displayNode =
     !livingCount || typeof livingCount !== 'object'
       ? undefined
-      : cloneElement(livingCount, oriProps => ({
+      : cloneElement(livingCount, (oriProps) => ({
```

Comment:
Hi 👋
And I found that `.prettierrc` (`"arrowParens": "avoid"`) and rome js config seem to be in conflict.

pre-commit will execute rome-format before